### PR TITLE
Add inventories payload to expvar

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Add invetories payload to expvar. This ease troubleshooting and developing.

### CODE REVIEW

**Review by ofek — DISMISSED** [review]

File: pkg/metadata/inventories/inventories.go [hunk 1]
```diff
@@ -149,15 +146,29 @@ func GetPayload(hostname string, ac autoConfigInterface, coll collectorInterface
 
 	agentCacheMutex.Lock()
 	defer agentCacheMutex.Unlock()
+	// Creating a copy of agentMetadataCache
+	agentMetadata := make(AgentMetadata)
+	for k, v := range agentMetadataCache {
```
Comment (truthbk) [inline pkg/metadata/inventories/inventories.go]: 🍰 

note to self: shallow copy should be fine.

File: pkg/metadata/inventories_collector.go [hunk 2]
```diff
@@ -19,34 +17,32 @@ const (
...
+	return inventories.GetPayload(hostname, c.ac, c.coll), nil
 }
 
 // Send collects the data needed and submits the payload
 func (c inventoriesCollector) Send(s *serializer.Serializer) error {
-	hostname, err := util.GetHostname()
+	payload, err := c.createPayload()
 	if err != nil {
-		return fmt.Errorf("unable to submit inventories metadata payload, no hostname: %s", err)
+		return nil
```
Comment (truthbk) [inline pkg/metadata/inventories_collector.go]: we don't want to propagate the error?

**Review by truthbk — COMMENTED** [review]
Nice, just a small question! Looks good otherwise.

File: pkg/metadata/inventories_collector.go (hunk 2 above)
Comment (hush-hush) [reply pkg/metadata/inventories_collector.go]: Good catch

**Review by truthbk — APPROVED** [review]

**Pull request merged**